### ISSUE TITLE
train loss about training_perception

### ISSUE THREAD

**Issue by Ahnhojin1223** [YvanYin/GoalFlow#36]
Hi, thank you for sharing GoalFlow!

I'm training the perception model using `run_goalflow_training_perception.sh`,  
but my training loss keeps oscillating and doesn't converge even after 40 epochs.  
(Attached TensorBoard screenshot shows losses for agent box/class and BEV semantic.)
<img width="1437" height="772" alt="Image" src="https://github.com/user-attachments/assets/eb18863d-f7d0-4f68-8977-75f7445b1426" />

Environment:
- 4× V100 (32GB, NVIDIA DGX)
- Batch size: 15 (to fully use VRAM)
- Epochs: 40
- Other configs: default

Questions:
1. Is this oscillating loss behavior expected, or am I missing any training setting (LR, warmup, grad clip, etc.)?
2. Why does the provided script use `batch_size=2`?  
   Does GoalFlow assume a specific global batch size or gradient accumulation setting?

Thanks for any advice!

**Comment by ZebinX** [comment]
1. The open-source code is the complete version, and the parameters in the shell script are already configured.
2. When I train on a 4090 GPU with batch_size=2, it already uses up all the VRAM.
Could you try reducing the [tf_d_model](https://github.com/YvanYin/GoalFlow/blob/411fdbda4e7b77874dbd38776de2b8b9408fbeaf/scripts/training/run_goalflow_training_perception.sh#L25C1-L25C31) size and see if that helps?
